build/deploy/README.md

```diff
@@ -0,0 +1,99 @@
+# Kubernetes deployment via Tanka
```

Comment:
@BenjaminPelletier Following use-case is not always guaranteed to be survived. 
For example, if we have these nodes in our cluster: [(n1, n2), (n3, n4), (n5,n6)]
Out of these n2, n4 and n6 are currently having replicated ranges. If I suspend n2 and n4 simultaneously, we lose some of the ranges.
<img width="800" alt="Screen Shot 2021-06-07 at 1 38 39 pm" src="https://user-images.githubusercontent.com/40968555/120956438-e3743e00-c796-11eb-8237-c7e720e9336e.png">
<img width="1184" alt="Screen Shot 2021-06-07 at 1 46 03 pm" src="https://user-images.githubusercontent.com/40968555/120956454-eb33e280-c796-11eb-820c-a7c063d0bf87.png">

Reply:
Good catch -- thanks.  Will fix.